bin/docsify

```diff
@@ -18,6 +18,7 @@ require("yargonaut")
   .errorsStyle("red.bold");
 
 const yargs = require("yargs")
+  .demandCommand(1, '# Please specify a command')
```

Comment:
Can you give a bit better message? Maybe something like this
Along with a different color.
```js
  .demandCommand(1,chalk.red("[ERROR] 0 argument passed. Please specify a command")).
```

else we can fallback to `yargs` 's default message.

Reply:
I would say to leave it to the default `demandCommand`